src/main/kotlin/com/zepben/auth/client/ZepbenTokenFetcher.kt

```diff
@@ -158,41 +212,21 @@ data class ZepbenTokenFetcher(
  * Helper method to fetch auth related configuration from `confAddress` and create a `ZepbenTokenFetcher`
  *
  * @param confAddress Location to retrieve authentication configuration from. Must be a HTTP address that returns a JSON response.
- * @param verifyCertificates: Whether to verify the certificate when making HTTPS requests. Note you should only use a trusted server
- *                           and never set this to False in a production environment.
  * @param authTypeField The field name to look up in the JSON response from the confAddress for `tokenFetcher.authMethod`.
  * @param audienceField The field name to look up in the JSON response from the confAddress for `tokenFetcher.authMethod`.
  * @param issuerDomainField The field name to look up in the JSON response from the confAddress for `tokenFetcher.authMethod`.
- * @param client HTTP client used to retrieve authentication configuration. Generated from `verifyCertificate` by default.
- * @param confCAFilename Filename of X.509 CA certificate used to verify HTTPS responses from configuration service. Leave as null to use system CAs.
- * @param authCAFilename Filename of X.509 CA certificate used to verify HTTPS responses from token service. Leave as null to use system CAs.
+ * @param confClient HTTP client used to retrieve authentication configuration. Defaults to HttpClient.newHttpClient().
+ * @param authClient HTTP client used to retrieve tokens. Defaults to HttpClient.newHttpClient().
  *
  * @returns: A `ZepbenTokenFetcher` if the server reported authentication was configured, otherwise None.
  */
 fun createTokenFetcher(
     confAddress: String,
```

Comment:
change: can you make this match the python SDK and be split into host, port, path properties, with only host being the required part?

Reply:
Defaulting to fetching config from `https://grpcdomain/auth` and ``https://grpcdomain/ewb/auth` will be done one layer up, in the sdk.